Makefile

```diff
@@ -151,7 +151,7 @@ ${PIP_SENTINEL}: ${VENV_SENTINEL} ${CONSTRAINTS_IN} ${SENTINEL}
 MXDEV_SENTINEL=${SENTINELFOLDER}pip-mxdev.sentinel
 ${MXDEV_SENTINEL}: ${PIP_SENTINEL}
 	@echo "$(OK_COLOR)Install mxdev$(NO_COLOR)"
-	@${PYBIN}pip install mxdev
+	@${PYBIN}pip install mxdev libvcs==0.11.1
```

Comment:
sorry ... needed for python>=3.9 otherwise you get `libvcs>=0.12.0` which is not compatible with mxdev